### PR TITLE
Add Max Quality Spot Analyzer

### PR DESCRIPTION
**Pull request by imagejan**

### CODE REVIEW

File: src/main/java/ch/fmi/MaxQualitySpotAnalyzer.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+	@Override
+	public boolean process() {
+		final SpotCollection sc = model.getSpots();
+
+		/*
+		 * First iteration to get max quality
+		 */
+		Iterator<Spot> spotIt = sc.iterator(frame, false);
+		double max = 0;
```
Comment (tinevez) [inline src/main/java/ch/fmi/MaxQualitySpotAnalyzer.java]: By convention the quality returned by detectors is always positive. 
But we don't know about future people respecting convention. I would initialise max with `Double.NEGATIVE_INFINITY` to be sure.

File: src/main/java/ch/fmi/MaxQualitySpotAnalyzer.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
+package ch.fmi;
+
+import java.util.Iterator;
+
+import fiji.plugin.trackmate.Model;
+import fiji.plugin.trackmate.Spot;
+import fiji.plugin.trackmate.SpotCollection;
+import fiji.plugin.trackmate.features.spot.SpotAnalyzer;
+
+public class MaxQualitySpotAnalyzer<T> implements SpotAnalyzer<T> {
```
Comment (tinevez) [inline src/main/java/ch/fmi/MaxQualitySpotAnalyzer.java]: Could you use the imglib2 formatter style?

Comment (imagejan) [inline src/main/java/ch/fmi/MaxQualitySpotAnalyzer.java]: Oh, I was using the ImageJ (SciJava) formatting style so far for my own projects.

Would you like `MaxQualitySpotAnalyzer` to be integrated into core TrackMate or your TrackMate-extras project? Then I am happy to apply whatever style you prefer. :-)

File: src/main/java/ch/fmi/MaxQualitySpotAnalyzer.java (hunk 1 above)
Comment (imagejan) [inline src/main/java/ch/fmi/MaxQualitySpotAnalyzer.java]: Good point. Will do.

**Pull request merged**